pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -137,6 +138,13 @@ type reconciler struct {
 	volumePluginMgr               *volumepkg.VolumePluginMgr
 	kubeletPodsDir                string
 	timeOfLastSync                time.Time
+	// TODO(xiangqian): An architecture change has been made in #26351 to move
+	// attach/detach from kubelet into controller manager since v.1.3. sig-storage
+	// is looking to completely disable kubelet volume attach/detach however the code
+	// still exists in kubelet. The following cache is needed to make sure the new
+	// changes do not break build, though needs to be cleaned up when attach/detach
+	// are completely removed from kubelet
+	operationStartTimeCache util.OperationStartTimeCache
```

Comment:
Or can it handle nil cache?

Reply:
adding checks in "VolumeAttach" call will make it capable of handling nil. Though IMHO, I'd rather make it explicit to avoid those unnecessary checks in the feature. Its an empty map which should not bring any burden to kubelet. I can add nil checking if its a more standard way of handling things in K8s though. WDYT?

Reply:
IIUC, for kubelet attach/detach, we don't set any start time, so the record is a no-op?

Reply:
that is right. Since kubelet workflow is a to-be-deprecated workflow, I think it makes sense to be a no-op? let me know if you think the otherwise